libnetwork/agent.go

```diff
@@ -225,11 +225,16 @@ func (c *Controller) agentSetup(clusterProvider cluster.Provider) error {
 	listen := clusterProvider.GetListenAddress()
 	listenAddr, _, _ := net.SplitHostPort(listen)
 
-	log.G(context.TODO()).Infof("Initializing Libnetwork Agent Listen-Addr=%s Local-addr=%s Adv-addr=%s Data-addr=%s Remote-addr-list=%v MTU=%d",
-		listenAddr, bindAddr, advAddr, dataAddr, remoteAddrList, c.Config().NetworkControlPlaneMTU)
+	log.G(context.TODO()).WithFields(log.Fields{
+		"listen-addr":               listenAddr,
+		"local-addr":                bindAddr,
+		"advertise-addr":            advAddr,
+		"remote-addr-list":          remoteAddrList,
+		"network-control-plane-mtu": c.Config().NetworkControlPlaneMTU,
+	}).Info("Initializing Libnetwork Agent")
```

Comment:
`dataAddr` isn't part of the log fields. Any specific reasons?

Reply:
Good eye; no, no reason (that I recall); I must've missed it when un-wrapping that one ☺️ 

I'll fix

Reply:
Also looks like data-path-port is not anywhere in this function, but I think that one is handled elsewhere, so that could've been "intentionally". (some cleaning up may still be needed in this area though)

Reply:
> Also looks like data-path-port is not anywhere in this function, but I think that one is handled elsewhere, so that could've been "intentionally". (some cleaning up may still be needed in this area though)

What do you mean? I don't see any `data-path-port` / `DataPathPort` anywhere :thinking: